### PR TITLE
Isolate Registration Feature

### PR DESCRIPTION
**Pull request by thomasogara**
### Registration Page
The Registration page now contains a form inviting an isolate to register with the service:
image

The form is inserted into the page using JSP, allowing for separation of concerns without relying on the client to fetch multiple assets after the initial page load.

### CODE REVIEW

File: src/main/webapp/isolate/registration.jsp [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Registration</title>
+</head>
+<body>
+  <div class="container" id="container">
+    <p>
+      Please fill out this form if you are someone who is isolating as a result
+      of COVID-19, and would like to be matched to a volunteer.
```
Comment (kevlartest) [inline src/main/webapp/isolate/registration.jsp]: I would rather state that they'd need help from a volunteer, as they'd probably be matched more than once

File: src/main/webapp/isolate/registration-form.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      placeholder="Last Name"
+      required
+      minlength="2"
+      maxlength="50"
+  >
+  <br>
+  <input
+      type="text"
+      name="address"
+      placeholder="Home Address"
```
Comment (kevlartest) [inline src/main/webapp/isolate/registration-form.html]: Maybe this should be isolation address? They might not be isolating at home

**Review by kevlartest — APPROVED** [review]

File: src/main/webapp/isolate/registration-form.html (hunk 2 above)
Comment (thomasogara) [reply src/main/webapp/isolate/registration-form.html]: yeah that sounds good, done https://github.com/googleinterns/step226-2020/pull/4/commits/e807cc16a046fc48a2bb5529d6df3bb78e4ef7cf

File: src/main/webapp/isolate/registration.jsp (hunk 1 above)
Comment (thomasogara) [reply src/main/webapp/isolate/registration.jsp]: https://github.com/googleinterns/step226-2020/pull/4/commits/6f8cfebff35c97ca3a285cced0d5b5512a45ce2f

**Pull request merged**